sdks/python/setup.py

```diff
@@ -189,7 +189,7 @@ def get_version():
     'google-cloud-bigquery>=1.6.0,<3',
     'google-cloud-core>=0.28.1,<2',
     'google-cloud-bigtable>=0.31.1,<2',
-    'google-cloud-spanner>=1.13.0,<2',
+    'google-cloud-spanner>=3.3.0',
```

Comment:
does it make sense to keep `<4` to avoid 'accidentally' upgrading a major version?

Reply:
we could! we just need the latest version of Spanner to ensure that the retries and timeout are accounted for.